pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -544,13 +544,15 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 							operatorHealthImpactLabelKey: "critical",
 						},
 					},
+					{
+						Record: "kubevirt_vm_count",
+						Expr:   intstr.FromString("sum by (namespace) (count by (name,namespace) (kubevirt_vm_error_status_last_transition_timestamp_seconds + kubevirt_vm_migrating_status_last_transition_timestamp_seconds + kubevirt_vm_non_running_status_last_transition_timestamp_seconds + kubevirt_vm_running_status_last_transition_timestamp_seconds + kubevirt_vm_starting_status_last_transition_timestamp_seconds))"),
+					},
 				},
 			},
 		},
 	}
 
-	// ((sum(kube_node_status_allocatable{resource='devices_kubevirt_io_kvm'}) or on() vector(0)) == 0 and (sum(kubevirt_configuration_emulation_enabled) or on() vector(0)) == 0) or (sum(kube_node_labels{label_kubevirt_io_schedulable='true'}) or on() vector(0)) == 0
```

Comment:
Leftover

Reply:
this is the removal of a leftover for a previous PR